arch/arm/boot/dts/nintendo3ds_ktr.dts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Support for:
+ *   - New Nintendo 3DS    (KTR-001)
+ *   - New Nintendo 3DS XL (RED-001)
+ */
+/dts-v1/;
+
+#include "nintendo3ds.dtsi"
+
+/ {
+	model = "Nintendo New 3DS (KTR)";
+	compatible = "nintendo,ktr", "nintendo,3ds";
+
+	fcram: memory@20000000 {
+		device_type = "memory";
+		reg = <0x20000000 0x10000000>;
```

Comment:
Until #1 is implemented, Linux really doesn't like this.  You can adjust these back to the CTR values, and this KTR dtb will boot.  Maybe I should just put those values in here now with a TODO on #1?

Reply:
Yeah the best thing will be to have linux ready to boot the ktr dts file any time now, enabling FCRAM is relatively easy